client/components/data/query-jetpack-sites-features/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
```

Comment:
I'd suggest adding some docs for this query component.

As simple as it is, I think it's paramount to take the time to explain and document the difference between `QuerySiteFeatures` and `QueryJetpackSitesFeatures`. The fact that we make a single request is much bigger than the obvious difference expressed in the site type.

Reciprocally, let's make sure to add a README for `QuerySiteFeatures` and stress on the difference there as well.